platform/darwin/docs/guides/For Style Authors.md.ejs

```diff
@@ -372,7 +372,7 @@ In style specification | Method, function, or predicate type | Format string syn
 `rgb`                  | `+[UIColor colorWithRed:green:blue:alpha:]` |
 `rgba`                 | `+[UIColor colorWithRed:green:blue:alpha:]` |
 <% } -%>
-`to-rgba`              | |
+`to-rgba`              | | `CAST(var, 'UIColor')` or `CAST(var, 'NSColor')`
```

Comment:
Within this particular .ejs file, you can use the `cocoaPrefix` variable to switch between “UI” and “NS” as appropriate:

https://github.com/mapbox/mapbox-gl-native/blob/e122fa43e04ecf89be0eafaf699c99e95be027bd/platform/darwin/docs/guides/For%20Style%20Authors.md.ejs#L5

We don’t have the “Predicates and Expressions” guide set up this way yet, since the platform-specific content is so minimal in that guide.